### PR TITLE
polymer++

### PR DESCRIPTION
**Pull request by notwaldorf**
`x-style` -> `custom-style`
`version++`
`polymer++`

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -19,16 +19,16 @@
...
-    "paper-ripple": "PolymerElements/paper-ripple#^0.8.0",
-    "paper-styles": "PolymerLabs/paper-styles#^0.8.0",
-    "paper-behaviors": "PolymerElements/paper-behaviors#^0.8.0"
+    "polymer": "Polymer/polymer#v0.9.0-rc.1",
+    "paper-ripple": "PolymerElements/paper-ripple#^0.9.0",
+    "paper-styles": "PolymerLabs/paper-styles#^0.9.0",
+    "paper-behaviors": "PolymerElements/paper-behaviors#^0.9.0"
   },
   "devDependencies": {
     "webcomponentsjs": "webcomponents/webcomponentsjs#~0.6.1",
```
Comment (cdata) [inline bower.json]: Should be `^0.6.0`

File: bower.json [hunk 2]
```diff
@@ -19,16 +19,16 @@
...
     "web-component-tester": "Polymer/web-component-tester#^2.2.3",
```
Comment (cdata) [inline bower.json]: Should be `*`

File: paper-radio-button.html [hunk 3]
```diff
@@ -43,7 +43,7 @@
 -->
 
 <dom-module id="paper-radio-button">
-  <style is="x-style">
+  <style is="custom-style">
     * {
       --paper-radio-button-unchecked-color: var(--primary-text-color);
       --paper-radio-button-unchecked-ink-color: var(--primary-text-color);
```
Comment (cdata) [inline paper-radio-button.html:49]: Inline defaults please!

**Comment by notwaldorf** [comment]
Done! (minus the inline defaults)

**Comment by cdata** [comment]
:+1:

**Pull request merged**